docs/source/workflow_inputs.rst

```diff
@@ -116,6 +118,19 @@ These features may require shorter timesteps, allow more sophisticated simulatio
   .. [#] DefrostModelType choices are "standard" and "advanced".
   .. [#] Use "standard" for default E+ defrost setting.
          Use "advanced" for an improved model that better accounts for load and energy use during defrost; using "advanced" may impact simulation runtime.
+  .. [#] OnOffThermostatDeadbandTemperature is currently only allowed with a 1 minute timestep.
+         Currently only supports homes with at most one cooling system (including heat pumps) serving 100% of the cooling load and at most one heating system (including heat pumps) serving 100% of the heating load (i.e., FractionHeatLoadServed and FractionCoolLoadServed are 1.0).
+         Applies to single speed and two speed air-source AC/ASHP only.
+  .. [#] An on/off thermostat deadband temperature is applied to both heating and cooling.
+         Typical values are between 2-4 F, with actual values being specific to the thermostat installed.
+         Note that thermostat deadbands are two sided.
+         As an example, if you had a heating setpoint of 71 F and a 2 F deadband, the heating equipment will turn on when the space temperature hits 70 F and off when it hits 72 F.
+         When this feature is enabled, the model will also explicitly model cycling, such that it will take several minutes for the HVAC to reach full capacity for single and two speed air-source AC/ASHP systems, and time-based realistic staging (stay at low speed for 5 minutes before transitioning to the higher stage, and stay at high speed until cut-out deadband temperature is reached) for two speed air-source AC/ASHP systems.
```

Comment:
I'm getting confused about when OnOffThermostatDeadbandTemperature applies. First, there are several references to "air-source AC/ASHP", which is redundant. ACs and ASHPs are by definition air-source. Second, I noticed that the changelog and BuildResidentialHPXML argument description both say "central AC/ASHP" instead, but there is a `base-hvac-room-air-conditioner-geb-onoff.xml` example file which suggests it does not apply to just central systems. Can you please clarify the language everywhere?

Reply:
Sorry for the confusion, it does apply to room ac as well, so I tried to correct it in the docs, missed the changelog and buildresidential arguments. I'm going to fix those too.